### DISCUSSION TITLE
Sandboxie-Plus 1.10.2 won't run Rpcss service and Dcomlaunch on latest Windows 10 Pro x64

### DISCUSSION THREAD

**Discussion by deeperfacts** [sandboxie-plus/Sandboxie#3146] [Q&A Feedback]
My OS is Windows 10 pro version 22H2 (OS Build 1904.3208)
with latest updates:

1.) exported as portable last night (no restart after exportation);
2.) run sandman.exe for the first time and it created the services;
3.) created new sandbox profile;
4.) tried to run a program today and it didn't work!

Errors: 

|Time|         |Message|                                                                                                                 

22:59:49.007   Sandboxie config has been reloaded (3)                                                                                    
23:01:05.706   SandboxieRpcSs.exe: SBIE2203 Failed to communicate with Sandboxie Service:  RPCSS_SXS - (Sandboxie RpcSs) [77/FFFFFFFF]   
23:01:05.706   SandboxieRpcSs.exe: SBIE2303 Could not hook PowerSettingRegisterNotification (1)                                          
23:01:08.102   Start.exe: SBIE2204 Cannot start sandboxed service RpcSs (1)                                                              
23:01:08.102   Start.exe: SBIE2204 Cannot start sandboxed service DcomLaunch (-4)                                                        
23:01:42.019   Sandboxie config has been reloaded (2)                                                                                    
23:02:08.343   SandboxieRpcSs.exe: SBIE2303 Could not hook PowerSettingRegisterNotification (1)                                          
23:02:08.390   SandboxieRpcSs.exe: SBIE2203 Failed to communicate with Sandboxie Service:  RPCSS_SXS - (Sandboxie RpcSs) [77/FFFFFFFF]   
23:02:21.729   Start.exe: SBIE2204 Cannot start sandboxed service RpcSs (1)                                                              
23:02:21.729   Start.exe: SBIE2204 Cannot start sandboxed service DcomLaunch (-4)                                                        
23:02:51.816   SandboxieRpcSs.exe: SBIE2303 Could not hook PowerSettingRegisterNotification (1)                                          
23:02:51.831   SandboxieRpcSs.exe: SBIE2203 Failed to communicate with Sandboxie Service:  RPCSS_SXS - (Sandboxie RpcSs) [77/FFFFFFFF]   
23:03:21.815   Start.exe: SBIE2204 Cannot start sandboxed service RpcSs (-1)                                                             
23:03:28.127   Start.exe: SBIE2204 Cannot start sandboxed service RpcSs (1)                                                              
23:03:28.127   Start.exe: SBIE2204 Cannot start sandboxed service DcomLaunch (-4)                                                        
23:03:58.239   SandboxieRpcSs.exe: SBIE2203 Failed to communicate with Sandboxie Service:  RPCSS_SXS - (Sandboxie RpcSs) [77/FFFFFFFF]   
23:03:58.239   SandboxieRpcSs.exe: SBIE2303 Could not hook PowerSettingRegisterNotification (1)                                          
23:04:01.514   Start.exe: SBIE2204 Cannot start sandboxed service RpcSs (1)                                                              
23:04:01.514   Start.exe: SBIE2204 Cannot start sandboxed service DcomLaunch (-4)                                                        
23:05:17.332   SandboxieRpcSs.exe: SBIE2303 Could not hook PowerSettingRegisterNotification (1)                                          
23:05:17.379   SandboxieRpcSs.exe: SBIE2203 Failed to communicate with Sandboxie Service:  RPCSS_SXS - (Sandboxie RpcSs) [77/FFFFFFFF]   
23:05:27.700   Start.exe: SBIE2204 Cannot start sandboxed service RpcSs (1)                                                              
23:05:27.700   Start.exe: SBIE2204 Cannot start sandboxed service DcomLaunch (-4)                                                        
23:05:57.774   SandboxieRpcSs.exe: SBIE2303 Could not hook PowerSettingRegisterNotification (1)                                          
23:05:57.899   SandboxieRpcSs.exe: SBIE2203 Failed to communicate with Sandboxie Service:  RPCSS_SXS - (Sandboxie RpcSs) [77/FFFFFFFF]   
23:06:00.411   Start.exe: SBIE2204 Cannot start sandboxed service RpcSs (1)                                                              
23:06:00.411   Start.exe: SBIE2204 Cannot start sandboxed service DcomLaunch (-4)                                                        
23:06:30.461   SandboxieRpcSs.exe: SBIE2303 Could not hook PowerSettingRegisterNotification (1)                                          
23:06:30.539   SandboxieRpcSs.exe: SBIE2203 Failed to communicate with Sandboxie Service:  RPCSS_SXS - (Sandboxie RpcSs) [77/FFFFFFFF]   
23:06:32.629   Start.exe: SBIE2204 Cannot start sandboxed service RpcSs (1)                                                              
23:06:32.629   Start.exe: SBIE2204 Cannot start sandboxed service DcomLaunch (-4)                                                        
23:07:02.615   Start.exe: SBIE2203 Failed to communicate with Sandboxie Service:  RPCSS_SXS - Start.exe [C0000034]                       
23:07:06.465   Starting troubleshooting script: SBIEMSG                                                                                  
23:07:11.069   yes: true                                                                                                                 
23:07:11.070   no: false                                                                                                                 
23:07:21.849   Troubleshooting script terminated                                                                                         

What is the problem?

**Comment by isaak654** [comment]
Duplicate of issue #3148
